jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -182,6 +184,7 @@ public class HiveConnection implements java.sql.Connection {
   private Subject loggedInSubject;
   private int maxRetries = 1;
   private IJdbcBrowserClient browserClient;
+  private Map<String, String> additionalHttpHeaders = new HashMap<String, String>();
```

Comment:
Was it necessary to promote this to a class member? It appears that usage is still limited to one method: `getHttpClient`.

Reply:
oh, right, makes sense, I'll change